fhir-config/src/main/java/com/ibm/fhir/config/FHIRConfiguration.java

```diff
@@ -36,8 +36,13 @@
     public static final String PROPERTY_CAPABILITY_STATEMENT_CACHE = "fhirServer/core/capabilityStatementCacheTimeout";
     public static final String PROPERTY_EXTENDED_CODEABLE_CONCEPT_VALIDATION = "fhirServer/core/extendedCodeableConceptValidation";
 
-    public static final String PROPERTY_SEARCH_PARAMETER_FILTER = "fhirServer/searchParameterFilter";
-
+    // Resources properties
+    public static final String PROPERTY_RESOURCES = "fhirServer/resources";
+    public static final String PROPERTY_FIELD_RESOURCES_INCLUDE_OMITTED = "includeOmittedResourceTypes";
```

Comment:
Paul wasn't a big fan of this property name, so we just had a chat and settled on `supportOmittedResourceTypes`:
```suggestion
    public static final String PROPERTY_FIELD_RESOURCES_SUPPORT_OMITTED = "supportOmittedResourceTypes";
```

Reply:
What about `open`as the property name? Where `open=true` allows inclusion of resources not explicitly defined in the configuration.

Reply:
Talked with Paul and we both like it.
```suggestion
    public static final String PROPERTY_FIELD_RESOURCES_OPEN = "open";
```